ibazel/main.go

```diff
@@ -26,6 +26,8 @@ var Version = "Development"
 
 var overrideableBazelFlags []string = []string{
 	"--action_env",
+	"--announce_rc",
+	"--bazelrc",
```

Comment:
Could you make a separate list for start-up flags?

Reply:
Yes, np.

Reply:
Added a new array for startup flags.